integration/tests/cook/test_basic.py

```diff
@@ -1707,3 +1714,15 @@ def test_user_limits_change(self):
         # reset user share fails (malformed) if no reason is given
         resp = util.reset_limit(self.cook_url, 'share', user, reason=None)
         self.assertEqual(resp.status_code, 400, resp.text)
+
+    def test_submit_with_no_name(self):
+        job_with_no_name = {'uuid': str(uuid.uuid4()),
```

Comment:
You should probably add a comment here to clarify that you need to manually set the `'uuid'` to avoid having the job name automatically set by the `submit_job` function.

Reply:
Good call, done